test/Main.hs

```diff
@@ -24,24 +24,6 @@ testFun expected = either
 
 main :: IO ()
 main = hspec $ do
-  describe "hasDate"
-    $ it "parses a legit date" (hasDate (BL8.pack "2/1") @?= Just (1, 2))
-  describe "The Karen Express" $ it
-    "parses a blob of JSON without error"
-    ( testFun
-        [ Menu (T.pack "Express")
-               (T.pack "Kycklinggryta, persilja, senap & kokt potatis")
-        , Menu (T.pack "Express - Vegan")
-               (T.pack "Vegetariska biffar, persiljekr\228m & potatismos")
-        ]
-
-    $ parse
-        "Swedish"
-        (BL8.pack
-          "{\"data\":{\"dishOccurrencesByTimeRange\":[{\"displayNames\":[{\"name\":\"Kycklinggryta, persilja, senap & kokt potatis\",\"categoryName\":\"Swedish\"},{\"name\":\"Chicken stew, parsley, mustard & boiled potatoes\",\"categoryName\":\"English\"}],\"startDate\":\"9/9/2019 12:00:00 AM\",\"dishType\":{\"name\":\"Express\"},\"dish\":{\"name\":\"Kycklinggryta, aubergin, zucchini, tomat & rostad potatis\"}},{\"displayNames\":[{\"name\":\"Vegetariska biffar, persiljekr\195\164m & potatismos\",\"categoryName\":\"Swedish\"},{\"name\":\"Vegetarian patties, parsley cream & mashed potatoes\",\"categoryName\":\"English\"}],\"startDate\":\"9/9/2019 12:00:00 AM\",\"dishType\":{\"name\":\"Express - Vegan\"},\"dish\":{\"name\":\"Vegetariska biffar, majonn\195\164s & potatispur\195\169\"}}]}}\n"
-        )
-    )
-
```

Comment:
While I agree with the premise of this PR, test for Kåren would still be good, one could convert it for current Kår resturangen

Reply:
Right, then I'll fix that.

I don't remember enough of this repo to know if Express was part of Kåren or not, but your absolutely right!

Reply:
I got there first! :D Is fix!